src/pages/app-development/index.md

```diff
@@ -1,75 +1,134 @@
 
-### Plugins and observers
+* **Add new payment gateways** - By integrating new payment gateways within the same process as the core application, businesses can ensure faster and more reliable payment processing, leveraging existing infrastructure for a smoother implementation.
 
-[Adobe I/O Events for Adobe Commerce](../events/index.md) enables building near real-time, event-driven Commerce integrations using App Builder. You can define key events, like customer account updates, to emit from Commerce and construct apps that listen and react to these events.
+* **Modify product display logic** - In-process adjustments to product display logic enable direct manipulation of the core application's data structures and functions, allowing for real-time updates to how products are presented, which can include adding of new custom attributes or PageBuilder sections.
 
-[Adobe Commerce Webhooks](../webhooks/index.md) enables configuring synchronous logic to execute calls from Commerce to external systems when key events occur. You can use this real-time communication with external systems to validate or modify data in Commerce.
+## Out-of-process extensibility
 
-### Admin development
+With out-of-process extensibility, custom code and extensions run in separate processes, often on different servers or services. This approach is increasingly popular with the rise of microservices and serverless architectures.
 
-The [Adobe Commerce Admin UI SDK](../admin-ui-sdk/index.md) enables an App Builder developer to extend the [Commerce Admin](https://experienceleague.adobe.com/docs/commerce-admin/start/admin/admin.html) to include custom menus and pages.
+The advantages of this approach include:
 
-### Custom cron jobs
+* **Isolation** - Custom code runs independently of the core application, reducing the risk of crashing or slowing down the main site.
+* **Scalability** - Out-of-process services can scale independently, allowing for better resource management and scalability.
+* **Technology agnostic** - Developers can use different programming languages or frameworks for their extensions, providing greater flexibility.
 
-App Builder uses Apache OpenWhisk Alarms to perform the scheduling services traditionally provided by cron jobs. [Scheduling Cron Jobs with Alarms](https://developer.adobe.com/app-builder/docs/resources/cron-jobs/) walks you through the process of implementing this feature.
+However, setting up and managing out-of-process extensions can be more complex, requiring additional infrastructure and communication protocols. For example, you might have additional database requirements, Elasticsearch set up, etc.
```

Comment:
Not sure why we have this line here since we provide the infrastructure (App Builder / API Mesh) to host the out of process extensions, Can we remove this and instead add a line something like "When choosing this out of process extensibility, Adobe provides developers with serverless runtime to host their apps reducing the overhead for developers"